src/transformers/models/yolos/feature_extraction_yolos.py

```diff
@@ -748,6 +748,61 @@ def to_tuple(tup):
             preds.append(predictions)
         return preds
 
+    # Copied from transformers.models.detr.feature_extraction_detr.DetrFeatureExtractor.post_process_object_detection
+    def post_process_object_detection(
+        self, outputs, threshold: float = 0.5, target_sizes: Union[TensorType, List[Tuple]] = None
+    ):
+        """
+        Converts the output of [`DetrForObjectDetection`] into the format expected by the COCO api. Only supports
+        PyTorch.
```

Comment:
I think the copied from statement needs to be removed over here:

Converts the output of [`YolosForObjectDetection`] into the format expected by the COCO api.

Reply:
Hmm I just actually add `with Detr->Yolos` after the `Copied from` statement, which will take care of appropriately adjusting the class name

Reply:
Ah, good to know!

Reply:
Opened a follow-up PR here: https://github.com/huggingface/transformers/pull/19687